windows/ReactNativeWebView/ReactNativeWebView.vcxproj

```diff
@@ -81,7 +81,9 @@
     <Import Project="$(ReactNativeWindowsDir)\PropertySheets\external\Microsoft.ReactNative.Uwp.CppLib.props" Condition="Exists('$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppLib.props')" />
     <Import Project="$(SolutionDir)\packages\$(WinUIPackageProps)" Condition="'$(WinUIPackageProps)'!='' And Exists('$(SolutionDir)\packages\$(WinUIPackageProps)')" />
   </ImportGroup>
-  <PropertyGroup Label="UserMacros" />
+  <PropertyGroup Label="UserMacros">
+    <DefineConstants Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(WinUIPackageVersion)','2.8.0-prerelease.210927001'))">WINUI_VERSION_COMPAT;$(DefineConstants)</DefineConstants>
```

Comment:
rename WINUI_VERSION_COMPAT to something like WINUI2_HAS_WEBVIEW

Reply:
actually probably better to use:
```suggestion
    <DefineConstants Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(WinUIPackageVersion)','2.8.0-prerelease.210927001'))">WINUI_VERSION_COMPAT;$(DefineConstants)</DefineConstants>
Collaborator
@asklar asklar 1 minute ago
rename WINUI_VERSION_COMPAT to something like WINUI2_HAS_WEBVIEW

 

actually probably better to use:
```suggestion

No file chosen
Attach files by dragging & dropping, selecting or pasting them.
  </PropertyGroup>
  <PropertyGroup />
  <ItemDefinitionGroup>
    <ClCompile>
@@ -113,6 +115,16 @@
      <PreprocessorDefinitions>NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
    </ClCompile>
  </ItemDefinitionGroup>
  <ItemDefinitionGroup Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(WinUIPackageVersion)','2.8.0-prerelease.210927001'))">
    <ClCompile>
      <PreprocessorDefinitions>WINUI_VERSION_COMPAT;%(PreprocessorDefinitions)</PreprocessorDefinitions>
    </ClCompile>
  </ItemDefinitionGroup>
  <ItemDefinitionGroup Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(WinUIPackageVersion)','2.8.0-prerelease.210927001'))">
    <Midl>
      <PreprocessorDefinitions>WINUI_VERSION_COMPAT;%(PreprocessorDefinitions)</PreprocessorDefinitions>
    </Midl>
  </ItemDefinitionGroup>
Commenting on lines +118 to +127
 

can replace this with:
```suggestion
<WinUI2_HasWebView2 Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(WinUIPackageVersion)','2.8.0-prerelease.210927001'))">true</WinUI2_HasWebView2>
```